nevergrad/functions/ml/mlfunctionlib.py

```diff
@@ -25,6 +25,50 @@ class MLTuning(ExperimentFunction):
     """Class for generating ML hyperparameter tuning problems.
     """
 
+    def get_dataset(self, dimension):
+
+        num_data: int = 120  # Training set size.
+        self.num_data = num_data
+        
+        # Training set.
+        X = np.arange(0., 1., 1. / (num_data * dimension))
+        X = X.reshape(-1, dimension)
+        random_state = np.random.RandomState(17)
+        random_state.shuffle(X)
+        y = np.sum(np.sin(X), axis=1).ravel()
+        self.X = X  # Training set.
+        self.y = y  # Labels of the training set.
+
+        # Variables for storing the cross-validation splits.
+        self.X_train = []  # This will be the list of training subsets.
```

Comment:
this needs to be initialized in the `init`, or at least typed in the init (without initialization)
same for X and Y
More generally speaking, all variables *need* to be expressed in the init so that typing and autocompletion work well.

Reply:
Ok!